src/main/kotlin/com/atlassian/performance/tools/virtualusers/api/load/ScenarioLoadProcess.kt

```diff
@@ -41,12 +41,8 @@ class ScenarioLoadProcess : LoadProcess {
         val behavior = options.behavior
         val scenario = behavior.scenario.getConstructor().newInstance()
         val browser = behavior.browser.getConstructor().newInstance()
-        val counter = ClusterNodeCounter()
```

Comment:
Ah yes, previously the counter was done once per process, now my "refactor" moved it to once per thread, that's why it cannot accumulate count above 1.

Reply:
Fixed.
The screenshot is a red herring